azure-pipelines.yml

```diff
@@ -35,7 +35,6 @@ jobs:
     enablePublishBuildAssets: true
     enableTelemetry: true
     helixRepo: aspnet/Extensions
-    timeoutInMinutes: 120 # increase timeout since BAR publishing might wait a long time
```

Comment:
This brings us back to the default of 1 hour correct?

Reply:
yes